### ISSUE TITLE
Attendees list items move down each time the fragment is resumed

### ISSUE THREAD

**Issue by ShridharGoel** [fossasia/open-event-orga-app#1506]
**Describe the bug**
Attendees list items move down each time the fragment is resumed.

**To Reproduce**
1. Go to Attendees.
2. Switch to some other app and come back or turn off the screen and then reopen.
3. RecyclerView items would shift down creating gaps.

**Expected behavior**
Position of items should not have changed.

**GIF**
![ezgif com-video-to-gif 40](https://user-images.githubusercontent.com/35566748/52347449-fbf9af80-2a47-11e9-80d5-4e6ae87a6f20.gif)

**Would you like to work on the issue?**
- [x] Yes
- [ ] No

**Comment by ShridharGoel** [comment]
Not able to figure out why this is happening.

@iamareebjamal @Masquerade0097 Can you help?

**Comment by iamareebjamal** [comment]
Will have to see, right now my suspect is the library we are using but not sure

**Comment by ShridharGoel** [comment]
Yes, most probably it's because of the sticky-headers-recyclerview library.

https://github.com/timehop/sticky-headers-recyclerview/issues/122

I've tried the possible fixes mentioned by people in the above thread but they don't seem to be working.

**Comment by iamareebjamal** [comment]
Use some other library

**Comment by ShridharGoel** [comment]
Okay.

**Comment by tavishjain** [comment]
Even I am facing this issue
